src/D2L.Bmx/ParameterDescriptions.cs

```diff
@@ -11,4 +11,7 @@ internal static class ParameterDescriptions {
 	public const string Output = "Custom path to the AWS credentials file";
 	public const string Format = "Output format of AWS credentials";
 	public const string NonInteractive = "Run non-interactively without showing any prompts";
+
+	public const string UseCache = "Enables Cache and specify time until the expiration " +
```

Comment:
Maybe mention this is for AWS

Reply:
The name use-cache doesn't immediately tell me that I need to provide a number with that so would be good to make sure we say what the number is doing